lib/public_api_v1.ts

```diff
@@ -124,6 +139,11 @@ app.post("/api/v1/debug", debugHandler);
 
 app.get("/api/v1/services/:serviceid", GetService(serviceModel));
 
+app.get(
+  "/api/v1/sender-services/:fiscalCode",
```

Comment:
this pattern doesn't work here, this means that `fiscalCode` is an identifier for the `sender-services` resource - in REST terms `fiscalCode` is a query pattern for retrieving `services` (the resource we're returning)

perhaps something like `/profile/:fiscalCode/senders` could work better or, if we want to avoid nested resources: `/services/for-recipient/:fiscalCode`